examples/python/easy_start_demo/README.md

```diff
@@ -0,0 +1,35 @@
+## easyDemo for using GRPC in Python
```

Comment:
gRPC is stylized with an initial lowercase character followed by three capital letters. We're not too strict in code, but we should be more careful in user-facing documentation.

Reply:
yes, I will change the word